eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -0,0 +1,122 @@
+# Note, due to how `Expand-Archive` is leveraged in this script,
+# powershell core is a requirement for successful execution.
+param (
+  # arguments leveraged to parse and identify artifacts
+  $ArtifactLocation, # the root of the artifact folder. DevOps $(System.ArtifactsDirectory)
+  $WorkDirectory, # a clean folder that we can work in
+  $ReleaseSHA, # the SHA for the artifacts. DevOps: $(Release.Artifacts.<artifactAlias>.SourceVersion) or $(Build.SourceVersion)
+  $RepoId, # full repo id. EG azure/azure-sdk-for-net  DevOps: $(Build.Repository.Id). Used as a part of VerifyPackages
+  $Repository, # EG: "Maven", "PyPI", "NPM"
+
+  # arguments necessary to power the docs release
+  $DocRepoLocation, # the location on disk where we have cloned the documentation repository
+  $Language, # EG: js, java, dotnet. Used in language for the embedded readme.
+  $DocRepoContentLocation = "docs-ref-services/" # within the doc repo, where does our readme go?
+)
+
```

Comment:
To help with debugging I suggest adding a line to output the command similar to what I added to other scripts see https://github.com/Azure/azure-sdk-tools/commit/e157cfa8c83cf55e65561b50d59f7cd6de5700e9.